x/exp/ast/node.go

```diff
@@ -67,6 +67,10 @@ type NodeTypeHas struct {
 	StrOpNode
 }
 
+type NodeTypeHasTag struct {
```

Comment:
Keep it one one line?

Reply:
Yes, done here: https://github.com/cedar-policy/cedar-go/pull/77/commits/36fd3e7573bac8103b345a0c8d413d6eef37dccb.